jormungandr-scenario-tests/src/main.rs

```diff
@@ -133,3 +133,67 @@ pub fn scenario_1(mut context: Context<ChaChaRng>) {
 
     controller.finalize();
 }
+
+pub fn scenario_2(mut context: Context<ChaChaRng>) {
```

Comment:
Could the name be more informative as to what the scenario tests?

Reply:
nop. they will be cleaned up when I have a better control of the different scenario and their execution